package.json

```diff
@@ -187,6 +187,31 @@
                 "when": "editorTextFocus && vim.use<C-[> && !inDebugRepl"
             },
             {
+                "key": "ctrl+w h",
+                "command": "extension.vim_navigateLeft",
+                "when": "vim.use<C-w>"
```

Comment:
these are all for when !editorTextFocus right? we should add !editorTextFocus

Because then the actions.ts ones cover when we are focused?